src/SDK/Client/Platform/methods/identities/register.ts

```diff
@@ -1,7 +1,11 @@
 import { Platform } from "../../Platform";
 import { wait } from "../../../../../utils/wait";
+import { createFakeInstantLock } from "../../../../../utils/createFakeIntantLock";
 import createAssetLockTransaction from "../../createAssetLockTransaction";
 
+// We're creating a new transaction every time and the index is always 0
+const ASSET_LOCK_OUTPUT_INDEX = 0;
```

Comment:
Let's move it to `createAssetLockTransaction` since it creates asset lock transactions.